cloud/wfm/src/executor/ExecutorImpl.ts

```diff
@@ -0,0 +1,37 @@
+import * as Promise from 'bluebird';
+import {Process} from '../process';
+import {ProcessInstance} from '../process-instance';
+import {Task, TaskEventData, TaskStatus} from '../task';
+import {Executor} from './index';
+import {InstanceRepository} from './InstanceRepository';
+
+export {InstanceRepository};
+export default class ExecutorImpl implements Executor {
```

Comment:
Missing comments. They are valuable information. 
We should explain this model this as if not now then probably never :P